tests/discover/libraries/test.sh

```diff
@@ -52,6 +52,16 @@ rlJournalStart
         rlAssertGrep 'the library is stored deep.' $tmp/output
     rlPhaseEnd
 
+    rlPhaseStartTest "strip_git_suffix"
+        rlRun "$tmt strip_git_suffix 2>&1 | tee $tmp/output" 0
```

Comment:
Side note - We don't need to execute all steps right? Issue happens in discover. If I'm not mistaken then all tests in this file can run just that step. On my slow net it is (just for strip_git_suffix) 16s vs 1m36s.
Of course this is PR on its own and not blocking this one.

Reply:
Thanks Lukas! You can definitely squash commits.
Yes, the issue is in the discover. All the tests in the file use `$tmt`.
`"tmt='tmt run -avvvddd plan --name'"`, where `-a` means `Run all steps, customize some.`
I wanted to take into account the structure of the test suite (=file) and used `$tmt`. I think the other tests in the file also don't use all the steps. But of course I can update line56:
`rlRun "$tmt strip_git_suffix 2>&1 | tee $tmp/output" 0` -> `rlRun "tmt run -vvvddd plan --name strip_git_suffix 2>&1 | tee $tmp/output" 0`

Reply:
I think it should be changed for everything, in separate PR.